tests/test_dataclasses.py

```diff
@@ -20,20 +20,27 @@
 from copy import deepcopy
 from dataclasses import asdict
 
-from pendulum import DateTime
+from pendulum import DateTime, datetime
 
 from d3a_interface.data_classes import (
     BidOfferMatch, BaseBidOffer, Offer, Bid, json_datetime_serializer,
     TradeBidOfferInfo, Trade, BalancingOffer, BalancingTrade, Clearing,
     MarketClearingState)
 from d3a_interface.utils import datetime_to_string_incl_seconds
 
+# pylint: disable=missing-function-docstring
+# pylint: disable=missing-class-docstring
+# pylint: disable=attribute-defined-outside-init
```

Comment:
Nit: this can be resolved using fixtures

Reply:
I like to skip this one this time. I promise if I am in this area again, will adapt it.

Reply:
Agreed ;)